Do not check the return code for MooseDocs check command

aeslaughter opened the pull request:
Checking the return code causes this test to fail with SQA documentation mistakes are
made, which isn't a problem with respect to the command. Thus, just run the command
and rely on the output checks to ensure that the command is working

(closes #18400)

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/781278/) on ff06728 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/18401/site)

This comment will be updated on new commits.

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit 901c9f1ba4e258fa7047a02812d3038f0dbc8e9e
+Subproject commit dbe1aaf543b86eae2193efa5e7d1508479a57692
```
**Caution!** This contains a submodule update

aeslaughter committed:
Do not check the return code for MooseDocs check command

Checking the return code causes this test to fail with SQA documentation mistakes are
made, which isn't a problem with respect to the command. Thus, just run the command
and rely on the output checks to ensure that the command is working

(refs #18400)

aeslaughter committed:
Add timeouts to check command tests to limit run time

(refs #18400)
